scale/job/test/execution/configuration/test_configurators.py

```diff
@@ -347,7 +380,26 @@ def test_configure_queued_job_ingest_without_new_workspace(self, mock_msg_mgr):
         from ingest.test import utils as ingest_test_utils
         scan = ingest_test_utils.create_scan()
         ingest = ingest_test_utils.create_ingest(scan=scan, workspace=workspace_1)
-        Ingest.objects.start_ingest_tasks([ingest], scan_id=scan.id)
+       
+        # start ingest_tasks (can't call method due to using command messages)
```

Comment:
same as above; you could split the execute method into sub methods where one does just what you want here

Reply:
Message executed!